WalletWasabi.Tests/UnitTests/SerializableExceptionTests.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using WalletWasabi.Models;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests
+{
+	public class SerializableExceptionTests
+	{
+		[Fact]
+		public void UtxoRefereeSerialization()
+		{
+			var message = "Foo Bar Buzz";
+			var innerMessage = "Inner Foo Bar Buzz";
+			string innerStackTrace = "";
```

Comment:
This should be also `var` to make it consistent

Reply:
Fixed.